dev-data/widget-communication.data.ts

```diff
@@ -20,6 +21,7 @@ export const communicationText = {
     discordLink: 'Дискорд сервер курса',
     firstParagraphSecondHalf:
       ', где вы можете посмотреть последние новости, задать вопросы и общаться со студентами.',
+    discordNote: 'Внимание! В некоторых странах для доступа к Discord требуется использование VPN. Если у вас возникают трудности с подключением, попробуйте использовать надёжный VPN- сервис.',
```

Comment:
```suggestion
    discordNote: 'Внимание! В некоторых странах для доступа к Discord требуется использование VPN. Если у вас возникают трудности с подключением, попробуйте использовать надёжный VPN-сервис.',
```
it looks like there is an extra space here

Reply:
maybe remove this dash at all like in the eng version?

Reply:
removed, thank you